--- Makefile.original	2023-09-04 02:45:26.330452221 +0200
+++ Makefile	2023-09-04 02:45:55.696963865 +0200
@@ -1,6 +1,6 @@
 
-WIN32GCC = /usr/local/gcc-4.8.0-qt-4.8.4-for-mingw32/win32-gcc/bin/i586-mingw32-gcc
-export SDKROOT = $(shell xcrun --sdk macosx --show-sdk-path)
+#WIN32GCC = /usr/local/gcc-4.8.0-qt-4.8.4-for-mingw32/win32-gcc/bin/i586-mingw32-gcc
+#export SDKROOT = $(shell xcrun --sdk macosx --show-sdk-path)
 
 all: nix
 
@@ -49,7 +49,7 @@
 	curl -sLO https://github.com/mrdudz/cc65-old/raw/master/cc65-sources-2.13.3.tar.bz2
 
 cc65-2.13.3/bin/cl65: cc65-sources-2.13.3.tar.bz2
-	tar zxf cc65-sources-2.13.3.tar.bz2
+	tar xvf cc65-sources-2.13.3.tar.bz2
 	(cd cc65-2.13.3; /usr/bin/sed 's!/usr/local!'${PWD}'/cc65-2.13.3!' <make/gcc.mak >Makefile; make -j4 bins || make bins && make install || true)
 
 c2t.h: c2t.h.0 makeheader mon/dos33.boot1.mon mon/dos33.boot2.mon asm/autoload.s asm/diskload2.s asm/diskload3.s asm/diskload8000.s asm/diskload9600.s asm/fastload8000.s asm/fastload9600.s asm/fastloadcd.s asm/inflate.s cc65-2.13.3/bin/cl65
